pkg/server/status.go

```diff
@@ -1002,13 +1002,16 @@ func (s *statusServer) LogFile(
 
 	// Read the logs.
 	reader, err := log.GetLogReader(req.File, true /* restricted */)
-	if reader == nil || err != nil {
-		return nil, fmt.Errorf("log file %s could not be opened: %s", req.File, err)
+	if err != nil {
+		return nil, errors.Newf("log file %q could not be opened: %s", req.File, err)
```

Comment:
This is still not quite right. This should be `errors.Wrapf(err, "log file %q could not be opened", req.File)`